engine/src/main/java/org/hibernate/search/query/dsl/sort/SortDistanceContext.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.query.dsl.sort;
+
+import org.hibernate.search.query.dsl.Unit;
+
+/**
+ * @author Emmanuel Bernard emmanuel@hibernate.org
+ */
+public interface SortDistanceContext extends SortOrder<SortDistanceContext>, SortAdditionalSortFieldContext, SortTermination {
+
+	/**
+	 * Hint for the backend to return data in the appropriate unit.
+	 */
+	SortDistanceContext in(Unit unit);
```

Comment:
I'm not sure what "to return data" is referring to. Shouldn't this be an option for projections?